Containers/README.md

```diff
@@ -0,0 +1,77 @@
+# Containers with .NET and Docker for Beginners
+
+Containers are a popular way of packaging and distributing applications in today’s Cloud Native landscape – but what are they, and how can .NET developers integrate them into their workflows? In this introductory series you’ll learn about Docker, Containers, and the tooling available in .NET to help make the whole process effortless!
+
+## Video Series Link
+
+Full Series:
+YouTube link: https://aka.ms/dotnet/beginnervideos/youtube/containers 
+
+Learn link: https://aka.ms/dotnet/beginnervideos/learn/containers
+
+Individual Video Links:
+1. [What's the deal with .NET & Containers?](https://youtu.be/HA8rpDWMRq0)
+1. [Making great containers with .NET, Visual Studio, and Visual Studio Code](https://youtu.be/qCxSYymD0ug)
+1. [Publishing your .NET containers everywhere!](https://youtu.be/21zduERRS3M) 
+
+## Overview
+
+In this video series, we will discuss Containers with .NET and Docker.
+
+| **Goal**              | Introduction to .NET Containers                                    |
+| ----------------------------- | --------------------------------------------------------------------- |
+| **What will you learn**       | Introduction to .NET containers, how to make great containers, and how to publish .NET containers everywhere                                        |
+| **What you'll need**          | [VS Code](code.visualstudio.com) or [Visual Studio](https://visualstudio.microsoft.com/downloads/), [C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit), [.NET SDK](https://dotnet.microsoft.com/en-us/download/visual-studio-sdks) |
```

Comment:
Can you add a link to get Docker as well? Users won't need it to _build_ containers, but the will need it to _run_ them, which the second and third videos do.

```suggestion
| **What you'll need**          | [VS Code](code.visualstudio.com) or [Visual Studio](https://visualstudio.microsoft.com/downloads/), [C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit), [.NET SDK](https://dotnet.microsoft.com/en-us/download/visual-studio-sdks) [Docker](https://docs.docker.com/get-docker/) |
```